src/main/scala/nl.knaw.dans.easy.stage/dataset/EMD.scala

```diff
@@ -79,17 +79,28 @@ object EMD extends DebugEnhancedLogging {
   }
 
   private def addPrivacySensitiveRemark(emd: EasyMetadata, agreementsXml: Elem): Unit = {
-    val signerId = agreementsXml \ "depositAgreement" \ "signerId"
-    val easyAccount = (signerId \@ "easy-account").toOption
-    val fullname = signerId.text
+
+    def userNamePart: String = {
+      val signerId = agreementsXml \ "depositAgreement" \ "signerId"
+      (signerId.text, (signerId \@ "easy-account").toOption, (signerId \@ "email").toOption) match {
+        case (fullName, None, None) => fullName
+        case ("", None, Some(email)) => email
+        case ("", Some(account), None) => account
+        case ("", Some(account), Some(email)) => s"$account ($email)"
+        case (fullName, None, Some(email)) => s"$fullName ($email)"
+        case (fullName, Some(account), None) => s"$account ($fullName)"
+        case (fullName, Some(account), Some(email)) => s"$account ($fullName, $email)"
+      }
+    }
+
+    def privacyPart(boolText: String): String = {
+      if (BooleanUtils.toBoolean(boolText)) "DOES"
+      else "DOES NOT"
+    }
 
     val remark = Option((agreementsXml \\ "containsPrivacySensitiveData").text) match {
```

Comment:
I know it was already this way, but would you mind taking a look at:

1. shouldn't this be `(agreementsXml \\ "containsPrivacySensitiveData").text.toOption`
2. would it be easier to change the pattern match to
    ```scala
    .map(containsPrivacySensitiveData => ...)
    .getOrElse {
        logger.warn(...)
        "..."
    }
    ```